vllm/worker/model_runner.py

```diff
@@ -78,10 +78,15 @@ def __init__(
         # cache in_wsl result
         self.in_wsl = in_wsl()
         self.kv_cache_dtype = kv_cache_dtype
+        self.is_neuron = is_neuron()
 
     def load_model(self) -> None:
-        self.model = get_model(self.model_config, self.device_config,
-                               self.lora_config)
+        if self.is_neuron:
+            self.model = get_model(self.model_config, self.parallel_config,
+                                   self.scheduler_config)
+        else:
+            self.model = get_model(self.model_config, self.device_config,
+                                   self.lora_config)
```

Comment:
Why do we have different function arguments for neuron `get_model`?

Reply:
The function arguments are different because:
1/ transformer-neuronx doesn't support lora inference yet.
2/ transformer-neuronx would require parallel_config and scheduler_config to compile the kernels.

This has been changed to `utils.get_model` that fetches the device-specific `get_model` dynamically.